examples/frameworks/flaskapp_gevent_kafka.py

```diff
@@ -0,0 +1,53 @@
+# Run with:
+#
+#   $ gunicorn flaskapp:app
+#
+#   pip install kafka-python gevent
+#
+###############################
+# Author : mohammed.vaghjipurwala@gmail.com
+# Version : 1.0
+# Description : Implementation of Kafka Consumer & Producer with
+#               Flask & gunicorn with gevent
+#
+# Blog: https://medium.com/@mohammed.vaghy/implementing-kafka-with-gunicorn-gevent-workers-87e679926ddc
+#
+###############################
+
+from flask import Flask
+from gevent import monkey
+from json import loads, dumps
+from kafka import KafkaConsumer, KafkaProducer
+
+app = Flask(__name__)
+
+monkey.patch_all(socket=False, thread=False)
+
+@app.route("/")
+def hello():
+    return "Hello World!"
+
+
+@app.route("/kafka/start-reading")
+def read_kafka():
+    producer = get_producer()
+    for message in get_consumer():
+        message = message.value
+        sent = producer.send('test_producer',
+                             json.dumps(data).encode())
```

Comment:
`data` is not bound here.

Reply:
`dumps(data)` and not `json.dumps(data)` since there is `from json import loads, dumps`